doc/api/fs.md

```diff
@@ -293,7 +293,7 @@ A class representing a directory stream.
 
 Created by [`fs.opendir()`][], [`fs.opendirSync()`][], or [`fsPromises.opendir()`][].
 
-Example using async interation:
+Example using async iteration:
```

Comment:
Optional nit, but I would remove this line entirely. The example code is obviously example code and does not need to be announced.